src/GradingStudents/GradingStudents.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+namespace App\GradingStudents;
+
+use InvalidArgumentException;
+
+class GradingStudents
+{
+    public function roundGrades($grades)
+    {
+        $grades = explode(PHP_EOL, $grades);
+        $students = (int) array_shift($grades);
+        $this->validateInput($students, $grades);
+
+        return implode(PHP_EOL, array_map(function ($grade) {
```

Comment:
>The function should return an integer array consisting of rounded grades.

O retorno também deveria ser um array, portanto o `implode()` aqui não é necessário. Você também pode adicionar type hints na assinatura da função.

Reply:
Novamente isso ficou confuso por conta da descrição da função =/
O dado para output é o seguinte:
>**Output Format**
For each grades[i], print the rounded grade on a new line.